docs/guides/src/main/server/db.adoc

```diff
@@ -30,35 +26,42 @@ The list of supported databases and their corresponding versions are:
 By default, the server uses the `dev-file` database. This is the default database that the server will use to persist data and
 only exists for development use-cases. The `dev-file` database is not suitable for production use-cases, and must be replaced before deploying to production.
 
+By default, the Keycloak server uses the `dev-file` database to persist data. This database is intended for development environments only. It is unsuitable for a production environment and must be replaced before deploying to production.
+
 == Configuring a database
 
-For each supported database, the server provides some opinionated defaults to make it simpler to configure a database. The defaults allow
-you to easily configure a database by just providing some key settings like the database host and credentials.
+For each supported database, the server provides some opinionated defaults to simplify database configuration. You complete the configuration by providing some key settings such as the database host and credentials.
 
-To configure the database vendor, run the `build` command to build a server image using the `db` build option to configure the server for specific database:
+.Procedure
```

Comment:
We are not using the `.Procedure` heading anywhere else in the guides, so please remove this